# 3.22.11.13

testing fast sync


error from mydt.py

![](2023-03-22-11-30-39.png)


![](2023-03-22-11-35-34.png)

![](2023-03-22-11-36-45.png)

[git issue](https://github.com/pedrolbacelar/Digital_Twin/issues/213)